app/code/core/Mage/Adminhtml/Block/Catalog/Product/Edit.php

```diff
@@ -250,6 +250,6 @@ public function getIsConfigured()
 
     public function getSelectedTabId()
     {
-        return addslashes(htmlspecialchars($this->getRequest()->getParam('tab')));
+        return addslashes(htmlspecialchars((string)$this->getRequest()->getParam('tab')));
```

Comment:
How about
```suggestion
        return addslashes(htmlspecialchars($this->getRequest()->getParam('tab', '')));
```

Reply:
Thanks. Rebased and applied your suggestion.